build-system/tasks/helpers.js

```diff
@@ -295,16 +295,21 @@ function compileMinifiedJs(srcDir, srcFilename, destDir, options) {
       endBuildStep('Minified', name, timeInfo.startTime);
     })
     .then(() => {
-      if (argv.fortesting && MINIFIED_TARGETS.includes(minifiedName)) {
-        return enableLocalTesting(`${destDir}/${minifiedName}`);
+      if (!argv.no_amp_config && MINIFIED_TARGETS.includes(minifiedName)) {
+        return applyAmpConfig(
+          `${destDir}/${minifiedName}`,
+          /* localDev */ !!argv.fortesting
+        );
```

Comment:
Maybe out of scope for this PR, but why are all these synchronous blocks chained in `then` calls?

Reply:
It's all legacy code that was written before `async` and `await` were a thing. IIRC, the blocks weren't always synchronous. I've filed #26562 to modernize `.then()` chains in `build-system/`.